EpiserverRedirects/System/SystemRedirectsEventsModule.cs

```diff
@@ -62,7 +62,7 @@ private void PublishedContentHandler(object sender, ContentEventArgs e)
             var lastVersion = ContentVersionRepository
                 .Service
                 .List(e.ContentLink)
```

Comment:
Suggestion: a 'List' method has an overload that accepts LanguageBranch, so maybe we can leverage that instead? Probably some micro-optimization because I assume that fewer data will be selected from DB and stored in memory :P

Reply:
Good catch!